front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -39,9 +43,15 @@ import org.springframework.web.bind.annotation.RestController
 @RequestMapping('pipelines')
 class PipelineController {
 
+  @Autowired(required = false)
+  PipelineTemplateDAO pipelineTemplateDAO = null;
+
   @Autowired
   PipelineDAO pipelineDAO
 
+  @Autowired
+  ObjectMapper objectMapper;
+
```

Comment:
While you're in here, could you switch the class to use an autowired constructor instead?

Reply:
Sure